### ISSUE TITLE
508-defect-3 [SCREENREADER]: Fine-tuning insert location component of facility locater

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#21765]
Label: 21053

    - [508-defect-3]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3

Issue Title: Defect 3, screen readers, fine-tuning insert location component of facility locater
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As an accessibility specialist who is also a screen reader user, I strive for the most seamless experience possible.
<h2>Details</h2>
To optimize this component for screen reader users, the following adjustments should be made:
<ul>
<li>The find my location component renders as a link. Since it does something rather than going somewhere, it should be a button.</li>
<li>When I clear my location I am presented with an error message, though I haven't had a chance to enter alternative content.</li>
<li>The use my location component is presented above the location edit box. Though this may be a matter of preference, I believe it is easier for screen reader users if it is presented below since we often begin navigating a search component by jumping to the first form field.</li>
</ul>

Acceptance Criteria

Environment

Steps to Recreate

Proposed Solution (if known)

WCAG or Vendor Guidance (optional)

Screenshots or Trace Logs

**Comment by mmiddaugh** [comment]
Related: 
- [Changes to improve "Use my location" screen reader experience](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31259) 
- [[DESIGN] Assess the placement and label for 'Use my location' ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27448)

**Comment by mmiddaugh** [comment]
will be addressed in [Changes to improve "Use my location" screen reader experience](https://github.com/department-of-veterans-affairs/va.gov-team/issues/31259)
